docker/image/app/Dockerfile.twig

```diff
@@ -17,7 +17,7 @@ RUN apt-get update \
 
 COPY .my127ws/docker/image/app/root /
 COPY .my127ws/docker/image/app/bin /usr/local/bin
-RUN chmod +x /usr/local/bin/helper
+RUN chmod +x /usr/local/bin/helper /lib/scripts/*
```

Comment:
git itself supports executable bit, would it make sense to use that instead?

Reply:
of course, but I was under the impression that executable bit would not carry over to the docker image when copied on the image file system?

Reply:
nope it copies the permissions

Reply:
excellent, TIL! I will simplify this then